dev-packages/browser-integration-tests/scripts/detectFlakyTests.ts

```diff
@@ -1,7 +1,30 @@
 import * as childProcess from 'child_process';
+import * as fs from 'fs';
 import * as path from 'path';
 import * as glob from 'glob';
 
+/**
+ * The number of browsers we run the tests in.
+ */
+const NUM_BROWSERS = 3;
```

Comment:
So, we previously set this to 4 but afaict at max, we test 3 browsers. Some tests potentially even only run on Chrome. So I think it's ok to reduce this in favour of more often running each test. My super unscientific testing says we currently run way shorter than 30min.